### DISCUSSION TITLE
Filament Fields Not Submitting Data in Livewire

### DISCUSSION THREAD

**Discussion by franckopheno** [filamentphp/filament#14344] [Help]
### Package

Form builder

### Package Version

V3

## Filament Fields Not Submitting Data in Livewire

I'm encountering an issue where Filament rich text editor, tag input, and select fields are not submitting data to the backend when used in a Livewire component. Despite filling out the fields, the data isn't being sent, and validation rules like required show that the fields are empty.

More details:

• I'm using Filament [version3], Livewire [3], and Laravel [10].
• I've tried refreshing the component, using alternative submission methods, and debugging the Livewire interaction, but the issue persists. 

I'm looking for help understanding if there are known issues with Filament fields in Livewire, recommended configuration options, or best practices for handling data submission in this scenario.

**Comment by leandrocfe** [comment]
Could you share the code?
